seastar/src/lib.rs

```diff
@@ -7,6 +7,7 @@ mod cxx_async_local_future;
 
 mod config_and_start_seastar;
 mod preempt;
+#[allow(dead_code)]
```

Comment:
Why is it needed?

Reply:
It turns out it isn't needed. I had *unused function* warning for all sleep functions. I don't have them now. I'm not sure what changed but it works now.